src/status_im/contexts/shell/activity_center/drawer/view.cljs

```diff
@@ -20,4 +20,9 @@
                                       :text       (i18n/label :t/notifications-marked-as-read
                                                               {:count unread-count})
                                       :theme      :dark})])
-                     (rf/dispatch [:hide-bottom-sheet]))}]]]))
+                     (rf/dispatch [:hide-bottom-sheet]))}
+       {:icon     :i/settings
+        :label    (i18n/label :t/chat-notification-preferences)
+        :on-press (fn []
+                    (rf/dispatch [:navigate-to :notifications-settings])
+                    (rf/dispatch [:hide-bottom-sheet]))}]]]))
```

Comment:
On the initial implementation, the `Notification settings` option is not added to the drawer (despite the Design showing the option) as the notification settings page is not designed/built yet.

The screen `notifications-settings` (which navigate to) is from legacy code. I'm not sure we need this option now.

Reply:
Okay, Will remove it.

Reply:
Done